website/src/content/api-reference/application-config.mdx

```diff
@@ -165,6 +165,13 @@ Note that the list can be extended at any time. _Last update 2022-01-01_.
 
 </Warning>
 
+<Info>
+
+You can use different `entryPointUriPath` values per deployment to mimic having multiple environments of the same Custom Application.
```

Comment:
Nit: I would maybe rephrase this a bit, to avoid wrong expectations.

```suggestion
Even though the `entryPointUriPath` is a unique identifier of a Custom Application, and thus should be defined statically in the source code, it can also be defined "dynamically" via environment variables using the environment placeholders.

This can be the case when having to deploy the Custom Application to different environments.
```

Reply:
Perhaps we want to link the "using environment placeholders" to [the section we explain this part](http://localhost:8000/api-reference/application-config#using-variable-placeholders)?

Reply:
Sure, good idea 👌

Reply:
Updated in this commit: f3ad4c9d5cf870b5787c36dcf77634b9f88b7051